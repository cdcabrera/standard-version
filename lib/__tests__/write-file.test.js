const fs = require('fs');
const writeFile = require('../write-file');

describe('write-file', () => {
  it('should attempt to write a file', () => {
    const mockWriteFile = jest.fn();
    const spy = jest.spyOn(fs, 'writeFileSync').mockImplementation(mockWriteFile);

    writeFile({ dryRun: true });
    writeFile({}, 'lorem/ipsum/dolor.sit', 'hello world');
    expect(mockWriteFile.mock.calls).toMatchSnapshot('write');

    spy.mockClear();
  });
});
