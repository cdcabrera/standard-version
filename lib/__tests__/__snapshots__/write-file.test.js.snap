// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`write-file should attempt to write a file: write 1`] = `
Array [
  Array [
    "lorem/ipsum/dolor.sit",
    "hello world",
    "utf8",
  ],
]
`;
